lib/model/model.go

```diff
@@ -65,6 +65,12 @@ type Availability struct {
 	FromTemporary bool              `json:"fromTemporary"`
 }
 
+type FileVersion struct {
+	VersionTime int64 `json:"versionTime"`
```

Comment:
We typically send times as `time.Time` over JSON. Why not here?

Reply:
We'd have to parse that there and back here requiring intermediate structs, and the API is based around timestamps.

Reply:
This is just a matter of `foo.versionTime = new Date(foo.versionTime)` on the Javascript side, and it should work automatically on the way back.